save-backend/src/main/kotlin/org/cqfn/save/backend/repository/ExecutionRepository.kt

```diff
@@ -1,10 +1,20 @@
 package org.cqfn.save.backend.repository
 
 import org.cqfn.save.entities.Execution
+import org.springframework.data.jpa.repository.Modifying
+import org.springframework.data.jpa.repository.Query
 import org.springframework.stereotype.Repository
 
 /**
  * Repository of execution
  */
 @Repository
-interface ExecutionRepository : BaseEntityRepository<Execution>
+interface ExecutionRepository : BaseEntityRepository<Execution> {
+    /**
+     * @param offset
+     * @param id
+     */
+    @Modifying
+    @Query("update execution set execution.offset = ?1 where execution.id = ?2", nativeQuery = true)
```

Comment:
We were trying to use `Sepcification`-based queries instead of native sql, have you considered using it here?

Reply:
The save () and saveAll() methods update the entity in the database.

Reply:
No, but now it's remade with typical save, so the method is not needed anymore